api/agent/agent.go

```diff
@@ -1051,11 +1065,17 @@ func newHotContainer(ctx context.Context, call *call, cfg *Config) (*container,
 	stderr := common.NewGhostWriter()
 	stdout := common.NewGhostWriter()
 
-	// these are only the conns, this doesn't write the syslog format (since it will change between calls)
-	syslogConns, err := syslogConns(ctx, call.SyslogURL)
-	if err != nil {
-		// TODO we could write this to between stderr but between stderr doesn't go to user either. kill me.
-		common.Logger(ctx).WithError(err).WithFields(logrus.Fields{"app_id": call.AppID, "path": call.Path, "image": call.Image, "container_id": id}).Error("error dialing syslog urls")
+	var syslogCon io.WriteCloser
+	if !cfg.EnableDockerSyslog {
```

Comment:
Same question about the conditional on `EnableDockerSyslog` here too.  Should this skip the not?  Am I misreading?